src/core/binding.c

```diff
@@ -1563,7 +1563,7 @@ QuicBindingReceive(
     //
 
     uint32_t Proc = CxPlatProcCurrentNumber();
-    uint64_t ProcShifted = ((uint64_t)Proc) << 40;
+    uint64_t ProcShifted = ((uint64_t)Proc + 1) << 40;
```

Comment:
I believe there are more places this needs to be updated.

Reply:
Ah, there was 1 more. Fixed.